ui/templates/hello.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>ISBN Website</title>
+
+    <style>
+      body {
+        font-family: arial, sans-serif;
+      }
+      h1 {
+        text-align: center;
+        margin-top: 100px;
+      }
+      .search {
+        text-align: center;
+        position: relative;
+        padding: 0;
+        margin: 0;
+      }
+      .searchbox {
+        width: 400px;
+        height: 40px;
+        background-color: transparent;
+        border: 1px solid grey;
+        border-radius: 10px;
+        margin-top: 30px;
+      }
+      .isbn {
+        display: flex;
+        margin-top: 10px;
+      }
+      .isbn-example {
+        background-color: #f8f9fa;
+        border: 1px solid #f8f9fa;
+        border-radius: 4px;
+        color: #3c4043;
+        font-size: 16px;
+        /* margin: 11px 4px; */
+        /* padding: 0 16px; */
+        line-height: 27px;
+        height: 46px;
+        min-width: 54px;
+        text-align: center;
+        cursor: pointer;
+      }
+      .flex {
+        margin-left: 50px;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+      }
+
+      .example {
+        margin-left: 420px;
+        margin-top: 40px;
+      }
+      .button {
+        margin-left: 8px;
+        margin-top: 30px;
+        width: 70px;
+        height: 43px;
+        border-radius: 10px;
+        border: 0px;
+        font-size: 18px;
+      }
+    </style>
+  </head>
+  <!-- 9788126568772 -->
+  <body>
+    <h1>ISBN Search Page</h1>
+    <div>
+      <div class="flex">
+        <input
+          type="text"
+          placeholder="Search.."
+          class="searchbox"
+          id="myForm"
+        />
+        <button onclick="myFunction()" class="button" id="myBtn">Go</button>
+      </div>
+      <div class="example">
+        <input
+          type="text"
+          value="9788126568773"
+          class="isbn-example"
+          id="example1"
+          onclick="exampleFunction1()"
+        />
+        <input
+          type="text"
+          value="9788126568772"
+          class="isbn-example"
+          id="example2"
+          onclick="exampleFunction2()"
+        />
+      </div>
+    </div>
+
+    <script>
+      function myFunction() {
+        var y = document.getElementById("myForm").value;
+        console.log(y);
+        window.location.href = " http://roost-worker:30045/books/" + y;
```

Comment:
Host of the endpoint should be read from ENV["API_ENDPOINT"]

Reply:
JS should make and API call to get book details. And JSON response returned from the API should be displayed in the same UI page at the bottom as Book Detail. (No redirection to http://$HOST/books/)

Reply:
ENV["API_ENDPOINT"] can be configured in it's container section of googlebookapi.yaml